supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -65,6 +113,185 @@
+
+*Use it*: yes
+
+*Incorrect forms*: Ceph Ansible
+
+*See also*:
+
+// Ceph: Added "In Red Hat Ceph Storage,"
+[discrete]
+[[ceph-mds]]
+==== image:images/yes.png[yes] ceph-mds (noun)
+
+*Description*: In Red Hat Ceph Storage, `ceph-mds` is the Metadata Server daemon. One or more instances of `ceph-mds` collectively manage the file system namespace, coordinating access to the shared OSD cluster. Always mark it correctly (`ceph-mds`)
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:metadata-server[Metadata Server], xref:mds[MDS]
+
+// Ceph: Added "In Red Hat Ceph Storage,"
+[discrete]
+[[ceph-mon]]
+==== image:images/yes.png[yes] ceph-mon (noun)
+
+*Description*: In Red Hat Ceph Storage, `ceph-mon` is the Ceph Monitor daemon. Always mark it correctly (`ceph-mon`).
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ceph-monitor[Ceph Monitor]
+
+// Ceph: Added "In Red Hat Ceph Storage,"
+[discrete]
+[[ceph-osd]]
+==== image:images/yes.png[yes] ceph-osd (noun)
+
+*Description*: In Red Hat Ceph Storage, `ceph-osd` is the Ceph object storage daemon that is responsible for storing objects on local file system and providing access to them over network. Always mark it correctly (`ceph-osd`).
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:osd[OSD], xref:object-storage-device[Object Storage Device],
+
+// Ceph: Added "In Red Hat Ceph Storage,"
+[discrete]
+[[ceph-radosgw]]
+==== image:images/yes.png[yes] ceph-radosgw (noun)
+*Description*: In Red Hat Ceph Storage, the `ceph-radosgw` daemon runs on Ceph Object Gateway nodes. Each instance provides a Civetweb web server and the object gateway functionality.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ceph-object-gateway[Ceph Object Gateway], xref:rados-gateway[RADOS Gateway], xref:rgw[RGW]
+
+// Ceph: Added "In Red Hat Ceph Storage,"
+[discrete]
+[[cephfs]]
+==== image:images/yes.png[yes] CephFS (noun)
+*Description*: In Red Hat Ceph Storage, CephFS is an initialization for the Ceph File System.
+
+*Use it*: yes
+
+*Incorrect forms*: cephfs
+
+*See also*: xref:ceph-file-system[Ceph File System]
+
+// RHEL: General; kept as is
+[discrete]
+[[certificate]]
+==== image:images/yes.png[yes] certificate (noun)
+*Description*: A certificate is an electronic document used to identify an individual, a server, a company, or other entity and to associate that identity with a public key. A certificate provides generally recognized proof of a person's identity. Public-key cryptography uses certificates to address the problem of impersonation.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:certificate-authorities[certificate authorities]
+
+// RHEL: General; kept as is
+[discrete]
+[[certificate-authorities]]
+==== image:images/yes.png[yes] certificate authorities (noun)
+*Description*: An entity that issues digital certificates. In Red Hat Identity Management, the primary CA is `ipa`, the IdM CA. The `ipa` CA certificate is one of the following types:
```

Comment:
Maybe the entry should be singular:
`A certificate authority (CA), is an entity that issues..`

For sentence completeness and to define the abbreviation